src/common/lockdep.cc

```diff
@@ -57,7 +57,7 @@ static bool free_ids_inited;
 static bool lockdep_force_backtrace()
 {
   return (g_lockdep_ceph_ctx != NULL &&
-          g_lockdep_ceph_ctx->_conf->lockdep_force_backtrace);
+          g_lockdep_ceph_ctx->_conf->get_val<bool>("lockdep_force_backtrace"));
```

Comment:
and remove `lockdep_force_backtrace` from `legacy_config_opts.h`. probably there are more of them.